### DISCUSSION TITLE
[GitHub] You have been blocked from flutter for 7 days

### DISCUSSION THREAD

**Discussion by PuzzleTakX** [community/community#70816] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi there 

[GitHub] You have been blocked from flutter for 7 days

I have been blocked for more than seven days, but my account has not been unblocked. Please follow up
My url account 
https://github.com/PuzzleTakX
Please  help me 🙏

**Comment by PuzzleTakX** [comment]
When I enter my profile page with another account, it gives a 404 error

**Comment by cehfisher** [comment]
Hi @PuzzleTakX - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!

**Comment by queenofcorgis** [comment, marked as answer]
Hi @PuzzleTakX ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
